xnmt/train.py

```diff
@@ -236,7 +237,7 @@ def one_epoch(self, update_weights=True):
         self.trainer.update()
 
       # Devel reporting
-      self.logger.report_train_process()
+      self.logger.report_train_process(self.trainer.optimizer.learning_rate)
```

Comment:
This argument is not important as such. I included it for debugging purposes to monitor the learning rate of the optimizer at every step.

Reply:
Why don't we pass in the trainer to logger at some point, so logger can save a reference to the trainer. Then if this reference is not None, logger will print out the learning rate in its logging information. I don't like adding an extra argument to this function, as it may make things complicated down the line.